translations/en.json

```diff
@@ -57,39 +49,21 @@
       "describe_phone_input_heading": "What is your telephone number?",
       "describe_mail_input_heading": "What is your e-mail address?",
       "describe_checkbox_input_description": "Yes, I give the example Municipality permission to forward my report to other organizations if the report is not intended for the Municipality.",
```

Comment:
Wellicht is het een idee om hier example Municipality te vervangen door een parameter die je meegeeft aan de translation en die betreffende parameter is dan de base.municipality uit config.json?

Reply:
oja, dat kan ook nog wel even snel met dit issue op zich, zit het er alvast in :)